package.json

```diff
@@ -36,12 +36,11 @@
     "karma-source-map-support": "1.2.0",
     "load-grunt-tasks": "3.5.2",
     "puppeteer": "1.3.0",
-    "request": "2.85.0"
-  },
-  "dependencies": {
+    "request": "2.85.0",
     "abp-filter-parser": "https://github.com/duckduckgo/abp-filter-parser/tarball/master",
     "btoa": "1.2.1",
     "tldjs": "2.3.1",
     "url-parse-lax": "^3.0.0"
-  }
+  },
```

Comment:
Moving all these to `devDependencies` because they're not needed for the Grade class itself. If they're in `dependencies`, we'd install them in the extension, and I don't think we want them double-installed.